docker/files/run/run.sh

```diff
@@ -9,10 +9,41 @@ set -ex
 # Helper functions #
 ####################
 
-function apply_configuration() {
+function catchup() {
+  local FAST_CATCHUP_URL="https://algorand-catchpoints.s3.us-east-2.amazonaws.com/channel/CHANNEL/latest.catchpoint"
+  local CATCHPOINT=$(curl -s ${FAST_CATCHUP_URL/CHANNEL/$NETWORK})
+  if [[ "$(echo $CATCHPOINT | wc -l | tr -d ' ')" != "1" ]]; then
+    echo "Problem starting fast catchup."
+    exit 1
+  fi
+
+  sleep 5
+  goal node catchup "$CATCHPOINT"
+}
+
+function start_public_network() {
   cd "$ALGORAND_DATA"
 
-  # check for config file overrides.
+  configure_data_dir
+  start_kmd &
+
+  if [ "$FAST_CATCHUP" = "1" ]; then
+    catchup &
+  fi
+
+  # redirect output to stdout
+  algod -o
+}
+
+function configure_data_dir() {
+  cd "$ALGORAND_DATA"
+  algocfg -d . set -p EndpointAddress -v "0.0.0.0:${ALGOD_PORT}"
+
+  if [ "$DEV_MODE" = "1" ]; then
+    algocfg -d . set -p EnableDeveloperAPI -v true
+  fi
```

Comment:
This setting is not specific to dev mode.
```suggestion
  algocfg -d . set -p EnableDeveloperAPI -v true
```

Reply:
It's useful often enough that most people would need to override the `config.json`, so maybe we need another environment variable for `DEV_API`.

Maybe the check here would be extended to `  if [[ "$DEV_MODE" = "1" || "$DEV_API" = "1" ]]; then`?

Reply:
For extra context, you can see a few API endpoints annotated with `EnableDeveloperAPI` on this page: https://developer.algorand.org/docs/rest-apis/algod/v2/#post-v2tealcompile